### DISCUSSION TITLE
Could this video sources be supported on query-streamlink

### DISCUSSION THREAD

**Discussion by C-Records** [iptv-org/community#502] [Q&A] [answered]
Hi, Im using query-streamlink for youtube streaming channels, and I wanted to know if could be possible to do the same with odysee.com and video.ibm.com. I was trying with the russian channel RT DE, which can be seen on odysee (blocked in the EU). I will leave the link below:

[https://odysee.com/@RTDE:e](https://odysee.com/@RTDE:e)

I got the following link, but it seems to be temporary:

[https://cdn.odysee.live/content/e23e889c6c0e06b3f59602061cee513067b506b6/master.m3u8](https://cdn.odysee.live/content/e23e889c6c0e06b3f59602061cee513067b506b6/master.m3u8)

About ibm, I tried with the live stream from the ISS (after the failure of the available m3u8 link).

[https://video.ibm.com/channel/iss-hdev-payload](https://video.ibm.com/channel/iss-hdev-payload)

I do not know if it's possible or how query-streamlink or another option could work in these cases.

Thank you very much for the effort you make in this project!

**Comment by BellezaEmporium** [comment, marked as answer]
Not quite sure how Odysee works yet, I'll have to look a bit further on it.

In the case of IBM Video, it is Ustream, and I believe they have a ustream plugin.

**Comment by Sphinxroot** [comment]
Odysee  .m3u8 will change over time if the stream stop

# https://odysee.com/@RT:fd/livestream_RT:d  16 days old
RT News
https://cdn.odysee.live/content/fdd11cb3ab75f95efb7b3bc2d726aa13ac915b66/master.m3u8

# https://odysee.com/@RTDE:e/livetv:e   16 days old
RT DE Live-TV
https://cdn.odysee.live/content/e23e889c6c0e06b3f59602061cee513067b506b6/master.m3u8

# https://odysee.com/@RTFRANCE:a/endirect:8  3+ months old
RT France
https://cdn.odysee.live/content/a6a125c99ba2a6c068a8b502fb08dda603f6f03a/master.m3u8

# https://odysee.com/@RTDocumentary:4/rtdlivestr:5  16 days old
RT Documentary
https://cdn.odysee.live/content/496730b88e41e74742fba5b6365435e5622dce92/master.m3u8

for ISS live  there a replacement https://www.youtube.com/watch?v=86YLFOog4GM
or 
https://www.youtube.com/results?search_query=iss+live&sp=EgJAAQ%253D%253D

**Reply by BellezaEmporium** [reply]
Seems rather easy to check at, if the stream, outside of given name has an ID that's in the middle of the URL.

**Reply by C-Records** [reply]
¿3 months? Wow, that's quite a while even for a dynamic link. I'll be aware of any progress regarding this platform.

About ISS live stream, that's an interesting option because of the ISS tracking map. Previous link, that shows the Ustream live channel, seems to be working again on: [http://iphone-streaming.ustream.tv/uhls/17074538/streams/live/iphone/playlist.m3u8](http://iphone-streaming.ustream.tv/uhls/17074538/streams/live/iphone/playlist.m3u8) But it's always good to have options.

Thanks for the information.

**Reply by HenkJanssen1980** [reply]
How can i check the new streams?

**Reply by BellezaEmporium** [reply]
Replace "cdn.odysee.live" to "cloud.odysee.live" with a referer pointed to https://odysee.com/

**Reply by HenkJanssen1980** [reply]
So i tried with link https://cloud.odysee.live/content/496730b88e41e74742fba5b6365435e5622dce92/master.m3u8 but this isn't working i'm affraid
